src/ocean_csys.cpp

```diff
@@ -126,6 +126,7 @@ double find_largest_root( const int ncoeffs, double* a ) {
     PolyDerivFunctor polyFunctor(a, degree);
```

Comment:
I bet there's a robust Boost function we could change to using and get rid of a bunch of code here. For v3.5?

Reply:
https://www.boost.org/doc/libs/1_64_0/libs/math/doc/html/math_toolkit/roots.html